components/brave_rewards/browser/test/rewards_contribution_browsertest.cc

```diff
@@ -355,9 +349,8 @@ IN_PROC_BROWSER_TEST_F(
 }
 
 // Check pending contributions
-IN_PROC_BROWSER_TEST_F(
-    RewardsContributionBrowserTest,
-    PendingContributionTip) {
+IN_PROC_BROWSER_TEST_F(RewardsContributionBrowserTest,
+                       DISABLED_PendingContributionTip) {
```

Comment:
They don't work since that information is no longer displayed in the rewards panel summary view. It may be added back though - see https://github.com/brave/brave-browser/issues/17869.